docs/installation/upgrade/include/2.3-to-2.4.inc

```diff
@@ -22,17 +26,60 @@ passdb some_name {
 }
 ```
 
+#### Empty userdb Variables
+
+userdb fields can be set to empty value now. Previously they became changed
+to `yes` value.
+
+#### Unknown/Invalid Variables
+
+Unknown/invalid `%{variables}` now cause Dovecot errors. THis may cause,
```

Comment:
THis

Also is there really a comma in here, looks weird?

Reply:
Lots of typos in there

Reply:
Oh, the commas around e.g. are correct.  It is meant to be by itself (at least in US English), so it needs to be separated from the rest of the sentence.